src/core/store/modules/app/app.actions.ts

```diff
@@ -117,6 +125,47 @@ const getUserAppData = createAsyncThunk<void, { network: Network; route: Route;
   }
 );
 
+/* -------------------------------------------------------------------------- */
+/*                                  Services                                  */
+/* -------------------------------------------------------------------------- */
+
+const checkServicesStatus = createAsyncThunk<void, void, ThunkAPI>(
+  'app/checkServicesStatus',
+  async (_arg, { dispatch, extra }) => {
+    const { YEARN_ALERTS_API } = extra.config;
+    try {
+      const servicesStatusResponse = await get(`${YEARN_ALERTS_API}/health`);
+      if (servicesStatusResponse.status !== 200) throw new Error('Service status provider not currently accesible');
+
+      const errorMessageTemplate =
+        'service is currently experiencing technical issues and have been temporarily disabled. We are sorry for the inconveniences, we are working towards issues been resolved soon.';
```

Comment:
The latter part of the sentence can be improved, maybe
```suggestion
        'service is currently experiencing technical issues and have been temporarily disabled. We apologize for any inconvenience this may have caused and our team is actively working on resolving these issues.';
```

Reply:
changed it here https://github.com/yearn/yearn-finance-v3/pull/674/commits/af9874b883993b37d88d3df4cd351647b1a091e3

close enough?

Reply:
Sounds better 👍